pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -773,6 +773,15 @@ def test_aggregate_mixed_types():
     tm.assert_frame_equal(result, expected)
 
 
+def test_aggregate_categorical_lost_index():
+    # GH: 28641 groupby drops index, when grouping over categorical column with min/max
+    ds = pd.Series(["b"], dtype="category").cat.as_ordered()
+    df = pd.DataFrame({"A": [1997], "B": ds})
+    result = df.groupby("A").agg({"B": "min"})
+    expected = pd.DataFrame({"B": ["b"]}, index=pd.Index([1997], name="A"))
+    tm.assert_frame_equal(result, expected)
+
+
 @pytest.mark.xfail(reason="Not implemented.")
```

Comment:
can you add the issue number below in the xfail message

Reply:
Done

Reply:
I meant IN the xfail message itself (e.g. the Not implementd, GH-3156)

Reply:
Done. Sorry I misunderstood you.